tests/test-recipes/metadata/_compile-test/f2_bad.py

```diff
@@ -1 +1 @@
-I am a bad file that should not pass compileall.
+"I am a bad file that should not pass compileall"
```

Comment:
```suggestion
I am a bad file that should not pass compileall.
```

Reply:
it is a `.py ` file it will fail to recognize the file itself... it wont matter what is inside the file..

Reply:
This is used for tests that compile `.py` to `.pyc` via `conda_build.post.compile_missing_pyc`.
`f2_bad.py` is intentionally a Python script with invalid syntax.
We test that `compile_missing_pyc` does not produce the corresponding `.pyc`  for it but also does not fail the build.